src/components/BlogPostListItem.astro

```diff
@@ -8,23 +8,24 @@ interface Props {
   link: string;
 }
 
-const { title, pubDate, description, author, tags, link } = Astro.props;
+const { title, pubDate, author, tags, link } = Astro.props;
 ---
 
 <li>
-  <a href={link}>
-    <article>
-      <time datetime={pubDate.toString()}>{pubDate.toDateString()}</time>
+  <article class="flex flex-col gap-y-2">
+    <time datetime={pubDate.toString()}>{pubDate.toDateString()}</time>
+    <a href={link} class="flex flex-col gap-y-2">
       <h3>{title}</h3>
-      <div>{author}</div>
-      <p>{description}</p>
```

Comment:
Yeah, not sure, for me it looked a bit cleaner without. Or maybe we just need to put a bit more thought into it.

I think we can still keep it on the blog posts, if im not wrong it can be used for [og](https://ahrefs.com/blog/open-graph-meta-tags/) tags as well, good for SEO. Not 100% though